catanatron_core/catanatron/models/board.py

```diff
@@ -78,6 +84,10 @@ def __init__(self, catan_map=None, initialize=True):
                 self.map.land_tiles.keys(),
             ).__next__()
 
+            # Cache buildable subgraph
+            global STATIC_GRAPH
+            self.buildable_subgraph = STATIC_GRAPH.subgraph(self.map.land_nodes)
```

Comment:
So... we now support MINI_MAP and some other map types. Does this work in those cases?